runtime/service_integ_test.go

```diff
@@ -295,6 +295,7 @@ func TestMultipleVMs_Isolated(t *testing.T) {
 							fmt.Printf("stderr output from vm %d, container %d: %s", vmID, containerID, stderrOutput)
 						}
 
+						require.NoError(t, exitStatus.Error())
```

Comment:
I'd suggest doing `require.NoError(t, exitStatus.Error(), "failed to retrieve exitStatus")`.

Reply:
Done